app/views/api/v1/account/admin/osbbs/show.json.jbuilder

```diff
@@ -0,0 +1,21 @@
+json.osbb do
+  json.id @osbb.id
+  json.name @osbb.name
+  json.phone @osbb.phone
+  json.email @osbb.email
+  json.website @osbb.website
+  json.address do
+    json.coordinates do
```

Comment:
`let src = "https://maps.googleapis.com/maps/api/staticmap?zoom=17&size=500x300&markers=size:small%7Ccolor:red%7C${this.state.osbb.address.latitude},${this.state.osbb.address.longitude}&key=AIzaSyB79BaHbHY_1floWCNQ1nlSx78zj6CYG04"`

does this work? if you have `address[:coordinates]` in response

Reply:
fixed